tests/e2e/utils/helpers.go

```diff
@@ -78,6 +78,7 @@ func GenerateRandomStringKeyValues(num int) []SimpleKeyValue {
 func newHTTPClient() http.Client {
 	doOnce.Do(func() {
 		defaultClient = http.Client{
+			Timeout: time.Second * 15,
```

Comment:
We have seen some calls taking over 1 minute on Windows. Shouldn't this timeout match the one below?

Reply:
I wanted to start with a sane value to see if it the timeouts are Dial/Connection or Read timeouts. Tests on this iteration passed so maybe the issues have been Dial/Connection related. I think we should re-trigger the tests a few times though to see what happens.

Reply:
So the issue we were having with windows networking was causing reads() to hang forever, hence why early runs of this were hitting context timeouts. Now that we've fixed this, we shouldn't expect reads to exceed 15s. This is a sane value.